app/src/common/constants/defectTypes.js

```diff
@@ -3,3 +3,11 @@ export const AUTOMATION_BUG = 'automation_bug';
 export const SYSTEM_ISSUE = 'system_issue';
 export const NO_DEFECT = 'no_defect';
 export const TO_INVESTIGATE = 'to_investigate';
+
+export const DEFECT_TYPES_SEQUENCE = [
+  PRODUCT_BUG.toUpperCase(),
+  AUTOMATION_BUG.toUpperCase(),
+  SYSTEM_ISSUE.toUpperCase(),
+  TO_INVESTIGATE.toUpperCase(),
+  NO_DEFECT.toUpperCase(),
```

Comment:
In `LaunchLevelEntities`, `SuiteLevelEntities` there's no localization keys for `NO_DEFECT`, and it leads to an error.

Reply:
Added